Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:14.04
 
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
this line is already there in install.sh. why is this required here?

Reply:
Yes, the above line is in install.sh but i am getting same warnings. So i put in docker file and i checked locally  its working